coding-concepts.js

```diff
@@ -21,26 +21,27 @@ const greeter = (name) => {
 }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Hello, "LEARN Students"
+// b) Verify and explain: I got undefined, now that I took more time ro read I see the reason. I think that is because in line 20 instead of name i
+// was supposed to write "LEARN Student" to get that result.
```

Comment:
You are getting there. You do get undefined. The reason is that there is a keyword missing....:)